sqlserver/check.py

```diff
@@ -84,6 +86,11 @@ def __init__(self, name, init_config, agentConfig, instances=None):
         self.connections = {}
         self.failed_connections = {}
         self.instances_metrics = {}
+        self.connector = init_config.get('connector', 'adodbapi')
```

Comment:
Beyond this?  This is what's currently in the example yaml:
 # Optional, change the connection method from adodbapi (the default) to
    # odbc (valid connector names are 'odbc' and 'adodbapi')
    # connector: odbc